release-26.2.1-rc: kv/kvclient/rangefeed: TestRangeFeedStartTimeExclusive failed

kv/kvclient/rangefeed.TestRangeFeedStartTimeExclusive [failed](https://mesolite.cluster.engflow.com/invocations/default/10734993-76c9-4a3b-b718-828fcc8e2c36?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZjbGllbnQvcmFuZ2VmZWVkOnJhbmdlZmVlZF90ZXN0) on release-26.2.1-rc @ [3dc9fd1adb98d3dc2cda24ecdba25c354c51a5b0](https://github.com/cockroachdb/cockroach/commits/3dc9fd1adb98d3dc2cda24ecdba25c354c51a5b0):


```
=== RUN   TestRangeFeedStartTimeExclusive
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestRangeFeedStartTimeExclusive1603292148
    test_log_scope.go:82: use -show-logs to present logs inline
    rangefeed_external_test.go:1327: -- test log scope end --
test logs left over in: outputs.zip/logTestRangeFeedStartTimeExclusive1603292148
--- FAIL: TestRangeFeedStartTimeExclusive (7.36s)
=== RUN   TestRangeFeedStartTimeExclusive/feed_type=mux/buffered_sender=false
    test_server_shim.go:168: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    test_server_shim.go:638: DRPC is enabled (via testserver factory defaults)
    rangefeed_external_test.go:1324: timed out waiting for event
--- FAIL: TestRangeFeedStartTimeExclusive/feed_type=mux/buffered_sender=false (4.52s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRangeFeedStartTimeExclusive.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-63808

--- a/pkg/kv/kvclient/rangefeed/rangefeed_external_test.go
+++ b/pkg/kv/kvclient/rangefeed/rangefeed_external_test.go
@@ -1320,7 +1320,7 @@ func TestRangeFeedStartTimeExclusive(t *testing.T) {
 			v, err := row.Value.GetInt()
 			require.NoError(t, err)
 			require.EqualValues(t, 3, v)
-		case <-time.After(3 * time.Second):
+		case <-time.After(testutils.SucceedsSoonDuration()):
 			t.Fatal("timed out waiting for event")
 		}
 	})

